src/monitor.rs

```diff
@@ -113,6 +113,14 @@ pub trait MonitorHandleProvider: AsAny + fmt::Debug + Send + Sync {
     #[cfg_attr(not(web_platform), doc = "detailed monitor permissions.")]
     fn position(&self) -> Option<PhysicalPosition<i32>>;
 
+    /// Returns physical size of monitor in millimeters.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **X11, **Wayland**, **MacOS**, **Windows**: Returns expected values.
+    /// - **Other platforms:** Always returns [`None`].
```

Comment:
Nit: Only note exceptions.

```suggestion
    /// - **iOS / Web / Android / Redox:** Unimplemented, always returns [`None`].
```